### PR TITLE
Create the basic structure of the MERN stack 

### PR DESCRIPTION
**Pull request by Jonathan204**
Set up a basic MERN stack using a Atlas: MongoDB. Basically all this can do is have a simple form that when submitted sends data to our db, and then using api calls we pull that data from the db into our project and display it.

### CODE REVIEW

**Comment by chris-mega** [comment]
Awesome!

File: client/src/reducers/posts.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+/* eslint-disable import/no-anonymous-default-export */
```
Comment (ColeVoelpel) [inline client/src/reducers/posts.js:1]: Should this comment be removed?

File: client/README.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/advanced-configuration](https://facebook.github.io/create-react-app/docs/advanced-configuration)
+
+### Deployment
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+
+### `yarn build` fails to minify
+
+This section has moved here: [https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify](https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify)
```
Comment (ColeVoelpel) [inline client/README.md:70]: I don't believe we should keep this file unless its useful

**Review by ColeVoelpel — COMMENTED** [review]
Looks great, I made some comments on things we might not need.

File: client/src/reducers/posts.js (hunk 1 above)
Comment (Jonathan204) [reply client/src/reducers/posts.js:1]: Yes it should I will remove that

**Pull request merged**